chat/src/main/java/io/skygear/plugins/chat/MessageCacheObject.java

```diff
@@ -0,0 +1,60 @@
+package io.skygear.plugins.chat;
+
+import android.support.annotation.NonNull;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Date;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+import io.skygear.plugins.chat.Message;
+import io.skygear.skygear.Record;
+
+public class MessageCacheObject extends RealmObject {
```

Comment:
I guess this is the object that is actually persisted to the realm store, a one-line comment to describe the class would be nice